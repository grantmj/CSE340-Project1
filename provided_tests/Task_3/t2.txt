TASKS 
    1 3

POLY 
    F = 5 x;
    G(x) = 7 x;

EXECUTE 
    INPUT x; 
    
INPUTS 
    0