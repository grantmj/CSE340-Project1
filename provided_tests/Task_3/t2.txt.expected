POLY - SORTED MONOMIAL LISTS
	F = 5 x;
	G(x) = 7 x;
